back/CinemaReservation.BusinessLayer/Contracts/IAccountService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Threading.Tasks;
+using CinemaReservation.BusinessLayer.Models;
+
+namespace CinemaReservation.BusinessLayer.Contracts
+{
+    public interface IAccountService
+    {
+        Task<AuthorizationResultModel> RegisterUserAsync(RegistrationModel registrationModel);
```

Comment:
It worth to use RegistrationResultModel here even if it has the same fields. The reason is statuses for registration differ from status for authorization